package.json

```diff
@@ -10,7 +10,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/Reggionick/s3-deploy.git"
+    "url": "git+https://github.com/Offlein/s3-deploy.git"
```

Comment:
no need to change the URL here.

Reply:
haha. That's embarrassing. I was having an issue testing from my local repo and trying a lot of stuff. Sorry about that.